catchup/service.go

```diff
@@ -41,6 +41,11 @@ import (
 const catchupPeersForSync = 10
 const blockQueryPeerLimit = 10
 
+// uncapParallelDownloadRate is a simple threshold to detect whether or not the node is caught up.
+// If a block is downloaded in less than this duration, it's assumed that the node is not caught up
+// and allow the block downloader to start N=parallelBlocks concurrent fetches.
+const uncapParallelDownloadRate = time.Second
```

Comment:
Could dynamic lambda reduce round times this far? There is probably a better value to use as the trigger.

Reply:
Looks like this only applicable to the follower node, a normal node would retrieve a block via agreement service. Maybe make sense to mention this like "applicable to nodes with agreement service disabled"

Reply:
@cce I guess thats an argument for something more sophisticated which could vary the parallel download with finer grain than all or nothing. Doesn't seem worth it to me.

Reply:
Maybe this should be set to the floor value off of dynamic lambda calculations?